elytron/WFLY-16306-keycloak-saml-feature-pack.adoc

```diff
@@ -0,0 +1,331 @@
+= WFLY-16306: Keycloak Galleon Feature Pack
+:author:            Farah Juma
+:email:             fjuma@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+It's currently possible to secure applications deployed to WildFly with SAML by installing the
+https://www.keycloak.org/docs/latest/securing_apps/#jboss-eap-wildfly-adapter-2[Keycloak Elytron SAML adapter].
+
+The general installation approach is to unpack the Keycloak adapter zip in the WildFly installation root directory.
+This installs some modules and provides some CLI scripts to complete the configuration. The CLI scripts can then
+be executed to install the `keycloak-saml` subsystem and update the default `application-security-domain` mapping in
+the `undertow` subsystem to use a newly installed `http-authentication-factory` which supports SAML.
+
```

Comment:
in a slight more abstract way that could be described as: `The CLI scripts registers the adapter into WildFly and adds the necessary configuration to use the "<auth-method>KEYCLOAK-SAML</auth-method>" in "web.xml" files;`; this description would really help someone my level, can we add something like this to the documentation?

Reply:
The description here of the CLI scripts was describing the old behaviour when using the Keycloak adapter zip.

In general though, the configuration that's done by the `keycloak-saml-layer` just makes it possible to secure an application with SAML. This can be done by either adding the `auth-method` in the web.xml file or by adding a `secure-deployment` resource to the `keycloak-saml` subsystem.